### PR TITLE
fix romanian pdf output + show how pdf's can be created

### PR DESCRIPTION
**Pull request by tudorbarascu**

### CODE REVIEW

**Comment by sorin-calinica** [comment]
Cool! Thank you!

File: readme.rst [hunk 1]
```diff
@@ -100,10 +100,10 @@ Sphinx intl extention ( https://pypi.python.org/pypi/sphinx-intl )::
 
     pip install sphinx-intl
 
-Then build::
+Then build:
```
Comment (yjacolin) [inline readme.rst]: `::` means to create a code block after a `:` This is not a typo.

**Comment by yjacolin** [comment]
@rduivenvoorde could you check the LaTeX stuff. I am not sure about this.

File: readme.rst (hunk 1 above)
Comment (tudorbarascu) [inline readme.rst]: @yjacolin 
Thanks for informing me, I wanted to show the user two options with bullets, and that was not code, so I deleted one of the ``:``

**Comment by tudorbarascu** [comment]
I successfully builded with the provided docker image so it should be safe.

**Comment by yjacolin** [comment]
See with @tudorbarascu, this is safe indeed.

**Pull request merged**